wp-content/themes/hurumap/functions.php

```diff
@@ -209,5 +209,42 @@ function bidirectional_acf_update_value( $value, $post_id, $field  ) {
 function custom_index_name() {
     return 'outbreak';
 }
+function get_base_url() {
+    $value = get_fields( 'options' );
+if (empty($value) ){
+    return $value;
+}else{
+    return "value";
+}
```

Comment:
Are we using this function anywhere? 

`get_fields(field, 'option')`

Reply:
@KhadijaMahanga  There is still a problem even when I do 
```
function get_base_url() {
    $value = get_field( 'frontend_base_url','options' );
    if ($value ){
        return $value;
    }else{
        return "";
    }
}
```